tests/iterate-embeddings.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+import sys
+
+from finalfusion import Embeddings
+
+if __name__ == "__main__":
```

Comment:
The unit tests should use pytest, they should not be standalone programs. Also, they should assert things.

Reply:
See the existing tests for examples.